mbed_app.json

```diff
@@ -450,8 +439,10 @@
             "storage.storage_type"                      : "TDB_INTERNAL"
         },
         "WIO_3G": {
+            "events.shared-eventsize"                   : 2048,
```

Comment:
no need to repeat the events here. Its already on top-section.

Reply:
Done.